### ISSUE TITLE
Cannot get email of invoice after order is confirmed

### ISSUE THREAD

**Issue by susankc** [it-projects-llc/odoo-saas-tools#831]
After all setup when i add a product in cart and registered all the process till wire transfer, i don't get email of Send SaaS creation page to partner Email template. And when i add manually want to send email invoice then following line is only executed
Now you can choose domain name for your service and create database  open following link

else link is not shown did i do something wrong ?

**Comment by susankc** [comment]
i have run sass in version 11 anybody to help ?

**Comment by ilmir-k** [comment]
@susankc 
Pull the latest updates and try to use ``saas_portal_subscription`` and ``saas_portal_subscription_sale`` modules

**Comment by susankc** [comment]
Hello @ilmir-k  i am in latest branch in version 11 but got same error is there any thing i need to setup to get link?

**Comment by ilmir-k** [comment]
Hello @susankc 
Note that partner will receive the link if you register payment for the corresponding invoice.
Another thing is you need to create a product and link it to SaaS plan first

**Comment by susankc** [comment]
I have test the all the process and at product i have link it to SaaS plan but automatic email after invoice is not send to partner i cant get anything what i do wrong ? @ilmir-k

**Comment by FFernandez-PlanetaTIC** [comment]
@susankc , have you installed `saas_portal_subscription_sale`?

**Comment by susankc** [comment]
Yes i have install this module and link my product to saas plan but when i confirm invoice automatic email is not send to user in order to create their database. @FFernandez-PlanetaTIC  @ilmir-k

**Comment by FFernandez-PlanetaTIC** [comment]
@susankc Email is sent on **pay** invoice
